apps/token-e2e/src/integration/flow/staking-flow.cy.js

```diff
@@ -199,22 +199,25 @@ context.skip('Staking Tab - with eth and vega wallets connected', function () {
       );
 
       cy.navigate_to('staking');
-      cy.get(stakeValidatorList).contains(this.validatorName).click();
+      // 1002-STKE-040
+      cy.click_on_validator_from_list(0);
 
+      // 1002-STKE-044, 1002-STKE-048
       cy.staking_validator_page_remove_stake('1');
 
+      // 1002-STKE-049
       cy.get(stakeNextEpochValue, epochTimeout)
         .contains(2.0, epochTimeout)
         .should('be.visible');
 
       cy.get(vegaWalletNextEpochBalances, txTimeout)
         .should('contain', 2.0, txTimeout)
-        .and('contain', this.validatorName)
+        .and('contain', partValidatorId)
```

Comment:
question (non blocking): Seems a odd to be asserting that the ellipsis is displayed in so many places? I might be missing something here.

Reply:
Originally there were kind of like placeholders for when we could get validators in Capsule with names but that probably won't happen anytime soon. I can also see there's a lot of repeated assertions for the epoch text. I've tidied that up so it's only asserting the texts once per element in the tests.